variables.tf

```diff
@@ -7,6 +7,11 @@ variable "project_id" {
   default     = null
 }
 
+variable "name" {
+  description = "The name of the VPN instance"
+  default = "openvpn"
```

Comment:
Remove the default. This should be required. The description should be `name to use when generating resources`